backend/lxd.go

```diff
@@ -33,16 +33,19 @@ var (
 	lxdImage         = "ubuntu:18.04"
 	lxdExecCmd       = "bash /home/travis/build.sh"
 	lxdDockerPool    = ""
+	lxdDockerDisk    = "10GB"
 	lxdHelp          = map[string]string{
 		"EXEC_CMD":       fmt.Sprintf("command to run via exec/ssh (default %q)", lxdExecCmd),
 		"MEMORY":         fmt.Sprintf("memory to allocate to each container (default %q)", lxdLimitMemory),
 		"CPUS":           fmt.Sprintf("CPU count to allocate to each container (default %q)", lxdLimitCPU),
 		"CPUS_BURST":     fmt.Sprintf("allow using all CPUs when not in use (default %v)", lxdLimitCPUBurst),
 		"NETWORK":        fmt.Sprintf("network bandwidth (default %q)", lxdLimitNetwork),
+		"POOL":           fmt.Sprintf("storage pool to use for the instances"),
 		"DISK":           fmt.Sprintf("disk size (default %q)", lxdLimitDisk),
 		"PROCESS":        fmt.Sprintf("maximum number of processes (default %q)", lxdLimitProcess),
 		"IMAGE":          fmt.Sprintf("image to use for the containers (default %q)", lxdImage),
 		"DOCKER_POOL":    fmt.Sprintf("storage pool to use for Docker (default %q)", lxdDockerPool),
+		"DOCKER_DISK":    fmt.Sprintf("storage pool to use for Docker (default %q)", lxdDockerDisk),
```

Comment:
could you update the description for this env var?

Reply:
Done